Ryujinx/Ui/VKRenderer.cs

```diff
@@ -72,7 +72,11 @@ protected override string GetGpuBackendName()
 
         protected override void Dispose(bool disposing)
         {
-            Device.DisposeGpu();
+            if (Device != null)
+            {
+                Device.DisposeGpu();
+            }
```

Comment:
```suggestion
                Device?.DisposeGpu();
```
Could this be written like that?

Reply:
True!